packages/linter/src/generators/init/init.ts

```diff
@@ -10,9 +10,9 @@ import {
 import {
   eslintConfigPrettierVersion,
   eslintVersion,
-  nxVersion,
   typescriptESLintVersion,
 } from '../../utils/versions';
+import { nxVersion } from '@nx/js/src/utils/versions';
```

Comment:
Why did this change?

Reply:
I thought it made sense to have nx as a central source of truth regarding nxVersion.

I'll revert that.